packages/venia-ui/lib/components/CheckoutPage/ItemsReview/showAllItemsFooter.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { mergeClasses } from '../../../classify';
+import Button from '../../Button';
+
+import defaultClasses from './itemsReview.css';
+
+const ShowAllItemsFooter = props => {
```

Comment:
I'd suggest reworking this component a bit.

- Make it just a button, rather than a button wrapped in a container
- Rename it to `ShowAllButton` or something
- Have it accept one prop, `onClick`
- Render a `button` element rather than our `Button` component
- Render the container element as part of the parent component instead

It should have its own CSS file. If it's just a button, it can have one classname: `root`.

Reply:
Got it. Done.